Forcibly disable the JIT on OpenBSD. The W+X mapping was rejected anyway
unless the binary using this library has an OPENBSD_WXNEEDED ELF
section, but avoids SIGABRT if the user has set kern.wxabort=1.

Index: src/qml/configure.json
--- src/qml/configure.json.orig
+++ src/qml/configure.json
@@ -103,7 +103,7 @@
                      && (config.linux || config.ios || config.tvos || config.qnx || config.integrity))"
             ],
             "output": [ "privateFeature" ],
-            "autoDetect": "!config.ios && !config.tvos",
+            "autoDetect": "!config.ios && !config.tvos && !config.openbsd",
             "comment": "On arm and arm64 we need a specialization of cacheFlush() for each OS to be
                         enabeled. Therefore the config white list.
                         Also Mind that e.g. x86_32 has arch.x86_64 but 32bit pointers. Therefore
